internal/bminventory/inventory.go

```diff
@@ -5066,6 +5077,8 @@ func (b *bareMetalInventory) V2UpdateHostIgnitionInternal(ctx context.Context, p
 			log.WithError(err).Errorf("Failed to parse host ignition config patch %s", params.HostIgnitionParams)
 			return nil, common.NewApiError(http.StatusBadRequest, err)
 		}
+	} else {
+		log.Warnf("Trying to apply empty host ignition config to host %s in infra-env %s", params.HostID, params.InfraEnvID)
```

Comment:
What is the side affect here? 
What will happen in this case?

Reply:
So what happens here is the following - as soon as you provide non-empty `params.HostIgnitionParams`, you enter this function. However, what we parse and apply is `params.HostIgnitionParams.Config`, meaning you have a potential field for confusion (that I hit when I was testing stuff). Imagine two scenarios

1) You have a host with ignition already overriden. You send a new override as `params.HostIgnitionParams` instead of `params.HostIgnitionParams.Config`. In reality, you have just overriden your previous ignition with an empty one. Not what was desired (and no warning on the way)

2) You have a host with no ignition. You send override as `params.HostIgnitionParams` instead of `params.HostIgnitionParams.Config`, as above. Once again, your override will not be applied.

The problem is that currently in both scenarios above, you will see the log message about successful update - `Custom discovery ignition config was applied to host %s in infra-env %s"`. This is confusing, because

* you see message saying override has been applied successfuly
* your host does not have any override

For this reason I decided to add a warning for a scenario when `params.HostIgnitionParams` exists but `params.HostIgnitionParams.Config` is empty.

Reply:
Seems like a valid use case (deleting an override) consider changing the log level to info.

Reply:
>Seems like a valid use case (deleting an override) consider changing the log level to info.

ACK